### ISSUE TITLE
Doesn't support schema stitching/merging resolvers

### ISSUE THREAD

**Issue by Akryum** [dotansimha/graphql-code-generator#1393]
**Describe the bug**
When using `mergeSchemas` from Apollo Server to stitch (merge) schemas together, the resolvers needs the `fragment` and `resolve` properties as [described here](https://www.apollographql.com/docs/graphql-tools/schema-stitching.html#adding-resolvers). The generated types don't appear to support this and typing errors are thrown.

**To Reproduce**
1. Stitch multiple schemas into one:

```js
const schema = mergeSchemas({
  schemas: [
    remoteSchema,
    localSchema,
  ],
  resolvers: mergeResolvers,
})
```

2. Use schema merging resolvers, for example:

```js
const mergeResolvers: IResolvers = {
  User: {
    apiKeys: {
      fragment: `... on User { id }`,
      resolve: ((user, args, ctx) => getUserApiKeys(user.id, ctx)),
    },
  },
}
```

3. Errors:

```
Type '{ fragment: string; resolve: (user: any, args: any, ctx: any) => Promise<any>; }' is not assignable to type 'Resolver<ApiKey[], User, Context, {}>'.
  Object literal may only specify known properties, and 'fragment' does not exist in type 'Resolver<ApiKey[], User, Context, {}>'.
```

```
Parameter 'user' implicitly has an 'any' type.
```

```
Parameter 'args' implicitly has an 'any' type.
```

```
Parameter 'ctx' implicitly has an 'any' type.
```

**Expected behavior**
- No errors
- Typing in `resolve`

**Schema/Documents**
Schema 1:

```graphql
type User {
  id: ID!
}
```

Schema 2:

```graphql
extend type User {
  apiKeys: [ApiKey!]!
}
```

**Environment:**
 - OS: elementary OS 0.4.1 (Ubuntu 16.04.5)
 - Codegen: 0.18.0
 - Node: 11.10.0

**Comment by Akryum** [comment]
Current workaround:

```ts
const mergeResolvers: IResolvers = {
  User: {
    apiKeys: {
      // @ts-ignore
      fragment: `... on User { id }`,
      resolve: ((user, args, ctx) => getUserApiKeys(user.id, ctx)) as UserResolvers.ApiKeysResolver,
    },
  },
}
```

**Comment by dotansimha** [comment]
@kamilkisiela can we fix it in the new ts-resolvers plugin?

**Comment by kamilkisiela** [comment]
@dotansimha of course!

**Comment by kamilkisiela** [comment]
@Akryum Does #1439 look good? I assumed `fragment` is always a `string` and `resolve` is a regular resolver function and required. Is that correct?

**Comment by Akryum** [comment]
That's correct! :)

**Comment by dotansimha** [comment]
Implemented by @kamilkisiela in the recent refactor, it will be available soon.

**Comment by dotansimha** [comment]
Fix available in 1.0.0 👍
